src/Compiler/Facilities/CompilerLocation.fs

```diff
@@ -365,7 +365,7 @@ module internal FSharpEnvironment =
             let path = location |> Option.defaultValue "<null>"
     #if DEBUG
             Debug.Print(sprintf """FSharpEnvironment.BinFolderOfDefaultFSharpCompiler (Some '%s') returned None Location
-                customized incorrectly: algorithm here: https://github.com/dotnet/fsharp/blob/03f3f1c35f82af26593d025dabca57a6ef3ea9a1/src/utils/CompilerLocationUtils.fs#L171"""
+                customized incorrectly: algorithm here: https://github.com/dotnet/fsharp/blob/03f3f1c35f82af26593d025dabca57a6ef3ea9a1/src/utils/CompilerLocation.fs#L171"""
```

Comment:
This link _may_ not work, since it's pointing to a specific commit, which doesn't have this file yet?

Reply:
Thanks!